UnitsNet/Scripts/UnitDefinitions/ForcePerLength.json

```diff
@@ -0,0 +1,35 @@
+﻿{
+  "Name": "ForcePerLength",
+  "BaseUnit": "NewtonPerMeter",
+  "XmlDoc": "Force per length is the amount of change applied per length.",
```

Comment:
Yes, my mistake.. I copied this from elsewhere but did not finish improving the text! I'll correct it